Makefile

```diff
@@ -23,8 +24,14 @@ deploy:
 	@$(IMAGE_BUILDER) run -it --rm -v /opt/cni/bin/:/opt/cni/bin/ $(IMAGE_NAME)
 
 generate-bin: generate
+	# Used in dockerfile
 	@cd userspace && go build -v
 
 generate:
+	# Used in dockerfile
 	for package in cnivpp/api/* ; do cd $$package ; pwd ; go generate ; cd - ; done
 
+rollout:
+	# Use sed to replace image name and then apply deploument file
+	sed "s|\(image:\).*\(#registory\)|\1 $(IMAGE_NAME) \2|g" ./kubernetes/userspace-daemonset.yml |kubectl apply -f -
```

Comment:
registry -> IMAGE_REGISTRY ?

Reply:
nvm, this check for "image: " and then for the end of sentence "#registry" and then adds the image_name in between them.